ITEM_en.tsv

```diff
@@ -4852,22 +4852,22 @@ ITEM_20150612_005238	A brush can clean the dusts
 ITEM_20150612_005239	Kraut Grass Juice
 ITEM_20150612_005240	A juice obtained from the Kraut grass. Monsters like it.
 ITEM_20150612_005241	Viscous Liquid
-ITEM_20150612_005242	A viscous liquid that allows the ink to settle in the tombstone.
+ITEM_20150612_005242	A viscous liquid that allows the faded ink on tombstones to appear again.
 ITEM_20150612_005243	Special Ink
 ITEM_20150612_005244	An ink obtained from Alchemist master.
 ITEM_20150612_005245	Rubbing of the tombstone. "You will wander due to the disconnection of destiny skein"
-ITEM_20150612_005246	Rubbing of the tombstone. "The capital's wall will go longer than the royal road."
+ITEM_20150612_005246	The rubbing on the tombstone says, "The capital's wall will go longer than the royal road."
```

Comment:
It should be "the rubbing OF the tombstone". A rubbing is when you take a sheet of paper, lay it against a surface, then literally rub charcoal (or similar substance) to more easily see engravings or other symbols of interest.